### ISSUE TITLE
phpMyAdmin logo

### ISSUE THREAD

**Issue by AghaSaad04** [phpmyadmin/phpmyadmin#15042]
### Is your feature request related to a problem? Please describe.

I think display logo thing should work even without actually applying the settings because then the user will come to know about how the layout will look without the logo.

### Describe the solution you'd like
Add the above functionality

### Additional context

- phpMyAdmin version: 4.8.5

![image](https://user-images.githubusercontent.com/36513474/54491568-5d335f00-48e1-11e9-8ae2-01c57ddad7c7.png)

after unchecking the display logo and applying:
![image](https://user-images.githubusercontent.com/36513474/54491598-8a800d00-48e1-11e9-95e1-d6869082a145.png)

the logo should be removed on runtime when the user unchecks the display logo so that he would have the idea how that layout would look like without logo before actually applying the settings.

![image](https://user-images.githubusercontent.com/36513474/54491659-e9de1d00-48e1-11e9-9260-16de7098a343.png)

**Comment by MrThanlon** [comment]
I think it's unnecessary, especially when the network is slow(considering RTT).

**Comment by MrThanlon** [comment]
I am going to contribute in GSoC 2019.
@williamdes, can you assign me this?
(BTW, how to add JavaScript to page? Which file I need to modify?

**Comment by mauriciofauth** [comment]
@MrThanlon It's not possible to assign issues to non-members. But you can open a pull request even if the issue is not assigned to you.

**Comment by MrThanlon** [comment]
#15051 @williamdes @mauriciofauth  I have solve it.

**Comment by MrThanlon** [comment]
#15052 solve it.

**Comment by mauriciofauth** [comment]
That's a good idea, but I don't know if applying this only to the logo is a good idea.
This can cause some confusion if only one of the settings has real-time changes and the other settings do not.
I think it would be more interesting that all settings take immediate effect rather than just a few.

**Comment by ibennetch** [comment]
I agree with @mauriciofauth, this change wouldn't be consistent with the existing behavior. I'd say at this point this isn't worth implementing.
